### ISSUE TITLE
Azure will give me two hundred dollars?

### ISSUE THREAD

**Issue by 1DTSzumski** [isaacs/rimraf#239]
You have to understand and take into consideration that I am half blind. $200 dollars on a hook like a piece of meat a dog chasing it dragging it through terrain that is full of holes snares traps. Fall into a financially unsustainable hole.  The feeding frenzy is like throwing meat in a pool full of puranas. Water boiling bones come to the surface. 
The bum rush hey they giving something away let's go. I almost came out of it with Baby Yoda capacity 2G or 5G actually affordable. Nope take that away my payment method was in place before the 5.40 bill was due.  My negative pessimistic view think that was not a deep enough hole so they offer me another $200 
I think they want me to get caught up in something more profitable for Azure. The $200 is an allusion, the definition of tangible is debatable. To see the forest through the trees. It is hell being half blind and half deaf. TIME is the only thing that is unescapable it catches up with all.

**Comment by 1DTSzumski** [comment]
Melt down after receiving an update odd the restart after the update the Dell collapsed immediately after restart.  Hard not to think that it was deliberate, but that is not helpful or solves anything. What bothered me, same stupid question what's your domain? Then the response that it does not exist as if it matters. I have noticed people have a different demeaner face to face. I am very Corgill and polite to a point.  The point, there is no point, that is the point. I partitioned the boot area of the disk made an error and last time everything was automatic restoring files my 2021 Pro plus. I have to get my files from SkyDrive and go try download the 2021 Pro plus and other things. Oh well live and learn?
